src/components/ResourceList/ResourcesList.js

```diff
@@ -73,49 +106,30 @@ class ResourcesList extends React.Component {
               resources
             </h1>
 
-            {this.props.data.allGetstartedJson.edges[0].node.data.map(
-              (category, idx) => (
-                <div
-                  key={idx}
-                  css={{
-                    marginBottom: 60
-                  }}
-                >
-                  <div
-                    css={{
-                      fontSize: 35,
-                      fontWeight: 300,
-                      color: colors.gray_100,
-                      padding: '33px 0 46px 0'
-                    }}
-                  >
-                    {category.title}
-                  </div>
-
-                  {category.data.map((entry, idx) => (
-                    <Card
-                      cssProps={{
-                        marginRight: 0,
-                        marginBottom: 24,
-                        ':nth-of-type(2n)': {
-                          marginRight: 24
-                        }
-                      }}
-                      key={idx}
-                      data={entry}
-                    />
-                  ))}
-                </div>
-              )
-            )}
+            {
+              this.props.resource ?
```

Comment:
I think you could pull this condition into the map

Reply:
correct, will do

Reply:
Actually, you can't break in array.map